floris/optimization/layout_optimization/layout_optimization_random_search.py

```diff
@@ -53,13 +53,26 @@ def _get_objective(
         layout_y,
         fmodel,
         yaw_angles=None,
-        use_value=False
+        use_value=False,
+        wind_resource_grid=None,
+        wind_resource_grid_wind_speeds = np.arange(0.0,25.0,2.0)
```

Comment:
There seems to be a lot of piping that has to happen to have this work; just for us to recommend using the HeterogeneousMap? Perhaps this and other comments around the two methods for layout optimization warrant some further discussion

Reply:
Definitely, do you have time today?

Reply:
This is all removed